db/migrate/20170128041649_change_survey_tables.rb

```diff
@@ -0,0 +1,43 @@
+class ChangeSurveyTables < ActiveRecord::Migration
+  def change
+    drop_table :course_survey_answer_text_responses do |t|
+      t.text :text_response
+    end
+
+    drop_table(:course_survey_question_text_responses) {}
+
+    add_column :course_survey_questions, :required, :bool, null: false, default: false
+    add_column :course_survey_questions, :question_type, :integer, null: false, default: 0
+    add_column :course_survey_questions, :weight, :integer, null: false
+    add_column :course_survey_questions, :max_options, :integer
+    add_column :course_survey_questions, :min_options, :integer
+    change_table :course_survey_questions do |t|
+      t.remove_actable index: { unique: true, name: :index_course_survey_questions_actable }
+    end
+
+    add_column :course_survey_answers, :text_response, :text
+    change_table :course_survey_answers do |t|
+      t.remove_actable index: { unique: true, name: :index_course_survey_answers_actable }
+    end
+
+    create_table :course_survey_question_options do |t|
+      t.references :question, null: false, foreign_key: { references: :course_survey_questions }
+      t.text :option
+      t.text :image
```

Comment:
@kxmbrian How about we make the option a rich text and directly put images there, then this field is no needed ? or you have other plans ?